examples/cluster/plot_dict_face_patches.py

```diff
@@ -33,9 +34,9 @@
 
 faces = datasets.fetch_olivetti_faces()
 
-# #############################################################################
+# %%
 # Learn the dictionary of images
-
+# ------------------------------
 print("Learning the dictionary... ")
 rng = np.random.RandomState(0)
 kmeans = MiniBatchKMeans(n_clusters=81, random_state=rng, verbose=True)
```

Comment:
This cell doesn't do any learning actually. It just defines what will be used.
I think it would make more sense to rename this cell "loading the data" and move the data loading into it